include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -1,8 +1,8 @@
 /**
- * @file cmaes.hpp
- * @author Marcus Edel
+ * @file cmaes_hoang.hpp
```

Comment:
```suggestion
 * @file cmaes.hpp
```

Use the correct filename.